doc/conventions.txt

```diff
@@ -1,25 +0,0 @@
-Here are the guidelines to follow when coding SuperTuxKart:
```

Comment:
Why delete this file?  It seems like this would be useful for first-time contributers...